### ISSUE TITLE
Example proposals aren't public

### ISSUE THREAD

**Issue by ericholscher** [psf/project-funding-wg#20]
It seems the example proposals listed here: https://github.com/psf/project-funding-wg/blob/master/projects.md aren't public. They are asking me to request access on Google. Is this intentional?

**Comment by xmunoz** [comment]
Yes! I asked for feedback on my [initial PR](https://github.com/psf/project-funding-wg/pull/18) about information that should be censored, but I did not receive any. If you don't see any cause for concern, feel free to make those links public.

**Comment by ericholscher** [comment]
Gotcha. I think @ejodlowska is probably the person to ask for official permission.

**Comment by ejodlowska** [comment]
It includes proposals written by the packaging-wg so I would not feel comfortable making that call without their approval. @di @ewdurbin?

**Comment by ewdurbin** [comment]
I think we would want to get approval from any parties who are named in each one before publishing, redacting anything pertinent to a specific person if requested.

**Comment by xmunoz** [comment]
Would you be ok with your personal information being public @ewdurbin? I have shared access to the docs where your information is present with you.

**Comment by brainwane** [comment]
@ewdurbin note - waiting for response from you on this.
